app/config/question-bank.js

```diff
@@ -2239,7 +2239,7 @@ const questionBank = {
           pageTitle: '',
           title: 'What type of slurry separator will you have?',
           hint: {
-            text: 'Capable to processing at least 3m3 of slurry per hour and producing at least 25% dry matter. '
+            text: 'Capable to processing at least 3m³ of slurry per hour and producing at least 25% dry matter. '
```

Comment:
This looks good based on ticket requirements, but isn't really good English

Nothing to change here, just wanted to raise it :)

Reply:
updated as per US